packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'DateTime'
+type: 'component'
+status: 'stable'
+slug: /components/date-time/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/datetime'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-datetime--basic'
+typescript: ./src/DateTime.tsx,./src/RelativeDateTime.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+In this package, there are two components (`DateTime` and `RelativeDateTime`) that will return a date following [Forma 36 guidelines](https://f36.contentful.com/guidelines/date-and-time/).
+
+## DateTime
+
+The DateTime component will format any date to a human friendly format and wrap it in a `<time>` tag.
+It allows four different forms of display that can be controlled by changing the `format` prop:
+
+| `format` option | Result                 |
+| --------------- | ---------------------- |
+| `full`          | 12 Aug 2020 at 8:00 AM |
+| `day`           | 12 Aug 2020            |
+| `time`          | 8:00 AM                |
+| `weekday`       | Wed, 12 Aug            |
+
+### Examples
+
+```tsx
+// import { DateTime } from '@contentful/f36-datetime';
+
+<DateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<DateTime date={new Date()} />; // passing a JS Date
+<DateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
+```
+
+All the usages above will result in 'Tue, 17 Aug 2021 at 3:45 PM'
+
+## RelativeDateTime
+
+Provides a relative date string for resolutions ranging from seconds up through years. Should be used when absolute date and time is unnecessarily specific.
+
+### Examples
+
+If we consider today as August 18th of 2021 and we use pass the following values to the Component
+
+```tsx
+// import { RelativeDateTime } from '@contentful/f36-datetime';
+
+<RelativeDateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<RelativeDateTime date={new Date()} />; // passing a JS Date
+<RelativeDateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
```

Comment:
This is still breaking on the documentation website, probably needs wrap the components with another component, and put the comments between `{}`, something like:
```tsx
<>
  <DateTime date="2021-08-17T15:45" /> { /* passing a Timestamp as an ISOString */ }
  <DateTime date={new Date()} /> { /* passing a JS Date */ }
  <DateTime date={1629240300000} /> { /* passing a Unix Epoch in milliseconds */ }
</>
```
but probably some other tag rather than `Fragment` to have spacing.

Also you can remove entirely the import statement, we removed the commented version from others components too.

Reply:
Ahhh of course! How did I forget about this? 🤦

Reply:
I fixed all of these bugs and also decided to split this into two page

![Screenshot 2021-08-20 at 17 56 36](https://user-images.githubusercontent.com/6597467/130260792-784f5934-5b8d-4976-a64a-adfd28b5e841.png)

Reply:
@gui-santos great idea! but for some reason, this change is not there yet? 🤔

Reply:
Really? It was my last commit

Maybe netlify is not re-deploying it

Reply:
Oh no! hahahaha
I forgot to commit my changes to gatsby-config! 🤦

Reply:
It should be fixed now